docs/translations/cn/cosmos-hub/launch/blog-2-en.md

```diff
@@ -0,0 +1,67 @@
+# The 3 Phases of the Cosmos Hub Mainnet
```

Comment:
this doesn't need to be on the chinese translations

Reply:
Done!